packages/wallets/declarations.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+    const content: React.SVGAttributes<SVGElement>;
+    export default content;
+}
```

Comment:
This resolves type errors warned in webpack due to importing SVGs as component using SVGR